src/main/webapp/app/shared/pages/imprint-page/imprint-page.component.scss

```
@@ -0,0 +1,35 @@
+.imprint-page {
+  color: var(--p-text-primary);
+  padding: 3rem 5rem 1.5rem 1.5rem;
```

Could you also please make the padding for the left and right sides equal so that the text is more centered 
```suggestion
  padding: 3rem 5rem 1.5rem 5rem;
```